pdf-rendering-template/bin/render-pdf.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * node.js script to render a given HTML file into the given PDF filename.
+ * Details of the API are at https://developers.google.com/web/tools/puppeteer/
+ *
+ * This command is invoked as:
+ *
+ *   $ node bin/render-pdf.js input.html output.pdf
+ *
+ */
+
+const puppeteer = require('puppeteer');
+
+const [, , rawInputPath, outputPath] = process.argv;
+const inputPath = `file://${rawInputPath}`;
+
+console.log('Input path:', inputPath);
+console.log('Output path:', outputPath);
+
+(async () => {
+  // Allow setting the path to Chrome via environment variable. This is usually
+  // not required but is useful for folks doing dev in WSL on Windows
+  const chromePath = process.env.PDF_RENDERING_PATH_TO_CHROME || undefined;
+
+  // `--no-sandbox` is required on Heroku - see
+  // https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-on-heroku
+  const browser = await puppeteer.launch({
+    args: ['--no-sandbox'],
```

Comment:
WSL will also most likely require `--single-process` , which is also typically required in low-memory envs.
Having the flag shouldn't have a negative impact on other platforms that don't require it, but also happy if you just want to stick it as a comment along the lines of `// you might also need --single-process` and circle back if it pops up as a problem down the road.

Reply:
Heroku is also a pretty low memory env so I think adding this seems good.